### ISSUE TITLE
[BUG] Broken older event pages

### ISSUE THREAD

**Issue by RiyaMathew-11** [fossunited/fossunited#593]
### Describe the bug

All of our past events in April except 1 - seem to have broken event pages now: 

https://fossunited.org/events/mangalore-april

Check ones in events timeline as well. 

It's caused mainly coz of this undefined variable (`user`) - as most of the speakers don't have field of pic urls then. 

```python
Traceback (most recent call last):
  File "apps/frappe/frappe/website/serve.py", line 20, in get_response
    response = renderer_instance.render()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/website/page_renderers/document_page.py", line 41, in render
    html = self.get_html()
           ^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/website/utils.py", line 523, in cache_html_decorator
    html = func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "apps/frappe/frappe/website/page_renderers/document_page.py", line 50, in get_html
    self.update_context()
  File "apps/frappe/frappe/website/page_renderers/document_page.py", line 65, in update_context
    ret = self.doc.get_context(self.context)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "apps/fossunited/fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py", line 149, in get_context
    context.speakers = self.get_speakers()
                       ^^^^^^^^^^^^^^^^^^^
  File "apps/fossunited/fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py", line 217, in get_speakers
    "profile_picture": cfp.picture_url or user.profile_photo or "/assets/fossunited/images/defaults/user_profile_image.png",
                                          ^^^^
NameError: name 'user' is not defined
```

**Comment by rahulporuri** [comment]
updated the PR description with the traceback

**Comment by rahulporuri** [comment]
closing as this issue got fixed in #602 . linting ftw!
